components/filemanager/overview.md

```diff
@@ -0,0 +1,423 @@
+                DateModifiedUtc = new DateTime(2022, 2, 3),
+                Path = Path.Combine(Documents.Path, "specification.docx"),
+                Size = 462 * 1024
+            };
+
+        var report = new FlatFileEntry()
+            {
+                Id = "5",
+                ParentId = Documents.Id,
+                Name = "Monthly report",
+                IsDirectory = false,
+                HasDirectories = false,
+                Extension = ".xlsx",
+                DateCreated = new DateTime(2022, 1, 20),
+                DateCreatedUtc = new DateTime(2022, 1, 20),
+                DateModified = new DateTime(2022, 1, 25),
+                DateModifiedUtc = new DateTime(2022, 1, 25),
+                Path = Path.Combine(Documents.Path, "monthly-report.xlsx"),
+                Size = 538 * 1024
+            };
+
+        #endregion
+
+
+        #region Images files coonfig
+        var dashboardDesign = new FlatFileEntry()
+            {
+                Id = "6",
+                ParentId = Images.Id,
+                Name = "Dashboard Design",
+                IsDirectory = false,
+                HasDirectories = false,
+                Extension = ".png",
+                DateCreated = new DateTime(2022, 1, 10),
+                DateCreatedUtc = new DateTime(2022, 1, 10),
+                DateModified = new DateTime(2022, 2, 13),
+                DateModifiedUtc = new DateTime(2022, 2, 13),
+                Path = Path.Combine(Images.Path, "dashboard-design.png"),
+                Size = 1024
+            };
+
+        var gridDesign = new FlatFileEntry()
+            {
+                Id = "7",
+                ParentId = Images.Id,
+                Name = "Grid Design",
+                IsDirectory = false,
+                HasDirectories = false,
+                Extension = ".jpg",
+                DateCreated = new DateTime(2022, 1, 12),
+                DateCreatedUtc = new DateTime(2022, 1, 12),
+                DateModified = new DateTime(2022, 2, 13),
+                DateModifiedUtc = new DateTime(2022, 2, 13),
+                Path = Path.Combine(Images.Path, "grid-design.jpg"),
+                Size = 1024
+            };
+
+        #endregion
+
+        var files = new List<FlatFileEntry>()
+            {
+                workFiles,
+
+                Documents,
+                specification,
+                report,
+
+                Images,
+                dashboardDesign,
+                gridDesign
+            };
+
+        return await Task.FromResult(files);
+
+    }
+}
+````
+
+
+## Data Binding
+
+The filemanager allows data binding to flat and hierarchical data. There are two alternative ways to provide data to the FileManager:
+
+* Set the FileManager Data attribute. In this case, the component will hold all the data.
+
+* Use the FileManager OnRead event and pass the data to the event argument. In this case, the FileManager will hold only the current set of data items.
```

Comment:
Maybe this will be clearer (assuming its correct):

```suggestion
* Use the FileManager `OnRead` event and pass the data to the event argument. In this case, the FileManager will hold only the data items for the current folder.
```